infra/testing/test-server.js

```diff
@@ -40,6 +40,13 @@ app.get(/\/__WORKBOX\/buildFile\/(workbox-[A-z|-]*)(\.(?:dev|prod)\.(.*))*/, (re
   res.sendFile(path.join(libraryPath, libraryFileName));
 });
 
+let eTagCounter = 0;
```

Comment:
I noticed that all the "helper" routes that used to exist in v2 have been removed from the test server.

@gauntface, are you cool with this being added to the test server? It greatly simplifies the test logic. If you'd rather see the route registered in a different way, I'm happy to refactor.

Reply:
It's fine with me - I just removed them as I was unsure as to what would be good and what wouldn't needed so please add what you want.